### ISSUE TITLE
Redirect URL 

### ISSUE THREAD

**Issue by iddnl07** [nodogsplash/nodogsplash#606]
I want to redirect to a URL without stopping the internet access.  I just want to display a page, like a survey,  and that's it. 
I would like this to happen when I try to connect to wifi. Can you help me with this ,please?

**Comment by bluewavenet** [comment]
@iddnl07 

> I want to redirect to an exter[n]al URL [] without stopping the internet access

You have two problems with this:

- Any captive portal requires the _**capture**_ of attempted Internet access to work, ie access is initially blocked.
- Almost all devices display the portal page, only while Internet access is blocked, closing the portal page as soon as Internet access is detected.

>  I just want to display a page, like a survey

This is problematic too:

- NoDogSplash cannot serve dynamic web pages or redirect to another service

If an initial blocking to enable the popup of specified web page(s) is acceptable, then you can serve dynamic web pages or redirect to an external server by using openNDS instead of NoDogSplash (see README.md)

**Comment by MarkVallevand** [comment]
So, NoDoSplash will not allow any redirection?  Not even if you poke holes using the firewall configuration of NDS?

**Comment by MarkVallevand** [comment]
I mean html redirection from the splash html, just to be clear.

**Comment by bluewavenet** [comment]
@MarkVallevand 

> So, NoDoSplash will not allow any redirection?

No, built in redirection was removed at the time of the split to openNDS.

> Not even if you poke holes using the firewall configuration of NDS?
> I mean html redirection from the splash html, just to be clear.
> 

Sure, this is how the original NoDogSplash FAS worked. It is very limited and easy to bypass though. It was made secure with an iptables redirect and the addition of encryption in openNDS.
